phpstan.neon

```diff
@@ -1,5 +1,6 @@
 parameters:
     checkMissingIterableValueType: false
     ignoreErrors:
+        - '#Parameter .* \$qualifier of function odbc_.* expects string, string\|null given\.#'
```

Comment:
V dokumentacii je, ze tam moze ist null, phpstanu sa to ale nepaci:
https://www.php.net/manual/en/function.odbc-tables.php